microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -74,6 +74,13 @@
     .bs-tooltip-auto[x-placement^=right] .arrow::before, .bs-tooltip-right .arrow::before {
         border-right-color: #fc8900 !important;
     }
+
+    .virtualPlaceholder {
+        display:block;
+        height:500px;
+        width:500px;
+        background:#eee;
+    }
```

Comment:
May have to mess with the sizing, the plotly plots tend to be wider than tall and do weird text cutoffs sometimes, but this is the right ballpark.

Reply:
Ya figured we'd have to poke it a bunch to get it to look right... but wanted to ahve some place holders in to eyeball how things look